test/pylib/util.py

```diff
@@ -141,3 +142,16 @@ async def get_enabled_features(cql: Session, host: Host) -> set[str]:
     """Returns a set of cluster features that a node considers to be enabled."""
     rs = await cql.run_async(f"SELECT value FROM system.scylla_local WHERE key = 'enabled_features'", host=host)
     return set(rs[0].value.split(","))
+
+def make_multy_exception_message(*exceptions : Exception):
```

Comment:
`*exceptions` is probably an array of exceptions, not an Exception? But maybe this weirdness is the correct Python typing syntax? Please try the type checker to see it doesn't complain about this syntax.

Reply:
The pep for annotations states that when using arbitrary argument lists, the expected type for **one** argument should be anotated:
https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values